cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -86,7 +89,16 @@ func (dm *deviceManager) ListAndWatch(empty *pluginapi.Empty, stream pluginapi.D
 }
 
 func (dm *deviceManager) Allocate(ctx context.Context, rqt *pluginapi.AllocateRequest) (*pluginapi.AllocateResponse, error) {
-	return deviceplugin.MakeAllocateResponse(rqt, dm.devices)
+	response, err := deviceplugin.MakeAllocateResponse(rqt, dm.devices)
+	// Set container annotations when programming is allowed
+	if dm.mode == devicecache.RegionMode || dm.mode == devicecache.RegionDevelMode {
```

Comment:
In the "regiondevel" mode the devices are supposed to be programmed from inside containers, not the CRI  hook.
The condition part `dm.mode == devicecache.RegionDevelMode` can be dropped.